CONTRIBUTING.md

```diff
@@ -1,109 +1,112 @@
+expected behavior that is not occurring.
 
-Before contributing to Velox, please review our coding style and best practices
-document in [`CODING_STYLE.md`](CODING_STYLE.md).
+Meta has a [bounty program](https://www.facebook.com/whitehat/) for the safe disclosure 
+of security bugs. In those cases, please go through the process outlined on that page 
+and do not file a public issue.
 
-## Code Formatting, Headers, and Licenses
+## Documentation
 
-Our Makefile contains targets to help highlighting and fixing format, header or
-license issues. These targets are shortcuts for calling `./scripts/check.py`.
+Help the community understand how to use the Velox library by proposing
+additions to our [docs](https://facebookincubator.github.io/velox/index.html) or pointing 
+out outdated or missing pieces.
 
-Use `make header-fix` to apply our open source license headers to new files.
-Use `make format-check` to highlight formatting issues using clang-format.
+## Code
 
-Formatting issues found on the changed lines in the current commit can be
-displayed using `make format-check`.  These issues can be fixed by using `make
-format-fix`. This command will apply formatting changes to modified lines in
-the current commit.
+This is the process we suggest for code contributions. This process is designed
+to reduce the burden on project reviews, impact on other contributors, and to
+keep the amount of rework from the contributor to a minimum.
 
-Header issues found on the changed files in the current commit can be displayed
-using `make header-check`. These issues can be fixed by using `make header-fix`.
-This will apply license header updates to files in the current commit.
+It is good to start with small bug fixes and tiny features to get familiar with
+the contributing process and build relationships with the community members.
+Look for GitHub issues labeled [good first issue](https://github.com/facebookincubator/velox/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or consider adding one of the
+[missing Presto SQL functions](https://github.com/facebookincubator/velox/issues/2262).
 
-An entire directory tree of files can be formatted and have license headers
-added using the `tree` variant of the format.sh commands:
-```
-    ./scripts/check.py format tree
-    ./scripts/check.py format tree --fix
+1. Sign the [Contributor License Agreement](https://code.facebook.com/cla) (CLA). This step needs to be completed only once.
 
-    ./scripts/check.py header tree
-    ./scripts/check.py header tree --fix
-```
+1. Review the [`LICENSE`](LICENSE) file. By contributing to Velox, you agree that your
+   contributions will be licensed under that LICENSE file. This step needs to be
+   completed only once.
 
-All the available formatting commands can be displayed by using
-`./scripts/check.py help`.
+1. Start a discussion by creating a Github issue, or asking on Slack (unless the change is trivial).
```

Comment:
Can we also pitch  the new discussions tab ?

Reply:
It is easy to convert an issue to a discussion, hence, this can be done anytime it becomes clear a discussion is better. Initially, I think it makes sense to start with an issue.